CNAME

```diff
@@ -1 +0,0 @@
-uwbhacks.com
```

Comment:
Rather than removing all of the files in the repo then adding everything back, I think migrating the files over will require being more selective. This file is very necessary for hosting on github pages with a custom domain name.